src/main/resources/import.sql

```diff
@@ -120,49 +46,49 @@ INSERT INTO startup_platform.investment_interests (id, category_id, sum, descrip
 --   ----------------------------------------------------------------------------------------------
 --  Fill projects table
 --   ----------------------------------------------------------------------------------------------
-INSERT INTO startup_platform.projects (id, name, owner_id, category_id, cost, date_added, date_start, min_invest, description, goal, stage)
+INSERT INTO startup_platform.projects (id, name, owner_id, category_id, cost, date_added, date_start, min_invest, description, goal, STAGE)
 VALUES
-  (1, 'Raccoon.World - gadgets for hands', 1, 1, 400000, '20.05.2017 14:30', '20.05.2017 15:30', 50000, 'Raccoon.World is a company that creates gadgets for hands, based on spatialization, tracking, reproduction of fine motor skills and tactile perception.
+  (1, 'Raccoon.World - gadgets for hands', 1, 1, 400000, '2017-05-20 14:30:30.069', '2017-05-20 15:30:30.069', 50000, 'Raccoon.World is a company that creates gadgets for hands, based on spatialization, tracking, reproduction of fine motor skills and tactile perception.
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal '2017-05-20 14:30:30.069' 10 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal '2017-05-20 15:30:30.069' 20 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)